tools/test

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-ROOT=$(dirname $0)/..
-$ROOT/tools/sigh test
-(cd $ROOT/extension && npm test)
-(cd $ROOT/shell && npm test)
+cd $(dirname $0)/..
+bazel test --noshow_progress --noshow_loading_progress --test_output=errors //particles/... //src/... //src_kt/...
```

Comment:
This and the next line seem like they could run in parallel, for future consideration. I assume `bazel test ...` doesn't need any of the Runtime TS built to JS?

Reply:
Do you mean something line this: https://github.com/PolymerLabs/arcs/blob/73d37c1ede10d0aaffc2902f3870db1192a3fbc1/.travis.yml#L28

I'll address this in a separate PR.

Reply:
I meant seeing if there's a way to run this and next command in parallel in Travis config, probably, like how you split the two build commands to build in parallel (I think?).

Reply:
Yes, we can split the bazel commands into multiple VMs. But, this might be complicated, since we'd need to make sure that the stuff gets copied to the test stage. 

I'll experiment.